analytics/views.py

```diff
@@ -1132,6 +1134,16 @@ def support(request: HttpRequest) -> HttpResponse:
             attach_discount_to_realm(realm, new_discount)
             msg = f"Discount of {realm.string_id} changed to {new_discount} from {current_discount} "
             context["message"] = msg
+        elif request.POST.get("new_subdomain", None) is not None:
+            new_subdomain = request.POST.get("new_subdomain")
+            old_subdomain = realm.string_id
+            if not Realm.objects.filter(string_id__exact=new_subdomain).exists():
```

Comment:
You can use`get_realm` and use try, except instead. Since we use `get_realm` everywhere else.

Also `string_id__exact=new_subdomain` and `string_id=new_subdomain` are both the same things.

Reply:
thanks for this! Today I learned this :)